### DISCUSSION TITLE
issue ssl certificate loading

### DISCUSSION THREAD

**Discussion by gajananmhaske** [OpenDDS/OpenDDS#3680] [Q&A] [answered]
Opendds_3.20/tests/DCPS/Messenger/

while running publisher and subscriber then getting error.

SSL::certificate::load: Warning: Unsported URI Scheme

failed to load documents supplied with URI - path

please help us 

thanks in advance

**Comment by mitza-oci** [comment, marked as answer]
I was not able to reproduce the error.  The test works as expected with both publisher and subscriber processes using `-DCPSConfigFile rtps_disc_sec.ini`.  If this is not the case for you, please include debug log files and steps to reproduce the problem.
